src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -67,6 +71,9 @@ public void should_store_a_PropertyAndSchoolAndRoom() {
         assertEquals(propertySaved.getRoomList().get(0).getRoomType(), propertyToSave.getRoomList().get(0).getRoomType());
         assertEquals(propertySaved.getRoomList().get(1).getRoomType(), propertyToSave.getRoomList().get(1).getRoomType());
 
+        assertNotNull(propertySaved.getFinancialData());
```

Comment:
It is good to check the values inside FinancialData and AnimalPolicy, not only check if they are not null

Reply:
done